cmd/restore/restore.go

```diff
@@ -55,12 +52,12 @@ func NewRestoreCmd() *cobra.Command {
 	}
 
 	cmd.SilenceUsage = true
-	cmd.Flags().StringVar(&restoredConfigPath, "config-out", "", "Specify desired name and full path for the restored k0s.yaml file (default: ${cwd}/k0s_<archive timestamp>.yaml)")
+	cmd.Flags().StringVar(&restoredConfigPath, "config-out", "/etc/k0s/k0s.yaml", "Specify desired name and full path for the restored k0s.yaml file (default: /etc/k0s/k0s.yaml)")
```

Comment:
Doesn't this default hint come automatically from cobra?

```suggestion
	cmd.Flags().StringVar(&restoredConfigPath, "config-out", "/etc/k0s/k0s.yaml", "Specify desired name and full path for the restored k0s.yaml file")
```

Reply:
fixed